.github/workflows/release-codecov-cli.yml

```diff
@@ -10,8 +10,12 @@ permissions:
   contents: read
 
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "hi from build and publish codecov-cli"
```

Comment:
hmmm

Reply:
lolol this is so the workflow doesn't break